stateoftheuniverse/widgets/sun.py

```diff
@@ -0,0 +1,39 @@
+import ephem
+
+
+# TODO:
+# * output formatting (timezones...) - in a separate file
+# * return times in a different format (ISO-8601 ?)
+# * it can be done in exactly same way for other ephem objects (Moon and planets) so we could
+#   abstract this functionality to a common module
+# * maybe use astroplan actually, pyephem is easy to use, but not always easy to install
+#   (C library) and has a bit bizarre design
+
+
+def get_sun_info(dt, lat, lon):
+    """
+    Get times of sunrise and sunset.
+
+    If the Sun is below the horizon, the next sunrise and sunset are returned.
+    If it's above the horizon - previous sunrise and next sunset.
+    
+    Args:
+        dt (datetime.datetime): datetime in UTC
+        lan (float): geographical latitude
+        lon (float): geographical longitude
+    Returns:
+        A tuple (sunrise time, sunset time) in UTC
```

Comment:
Could it return a datetime for both just to avoid any possible confusion? Not necessary if you think it makes the output less parseable, I think!

Reply:
This is one of the issues with pyephem - methods `next_rising` etc return pyephem custom format and while converting it is easy, maybe it wouldn't be necessary with astroplan.

Reply:
Oh okay that makes sense, we definitely want to avoid weird custom formats (y)